index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/brands.min.css" integrity="sha512-lCU0XyQA8yobR7ychVxEOU5rcxs0+aYh/9gNDLaybsgW9hdrtqczjfKVNIS5doY0Y5627/+3UVuoGv7p8QsUFw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style/style.css">
+    <title>Portifolio - Figma layout</title>
+    
+</head>
+<body>
+    <div class="top-mob">
+        <header>
+            <nav>
+                <a class="logo" href="#">Willian.T</a>
+                <a href="#"><img class="button-mobile" src="img/mobile-button.pnggit" alt="mobile-icon"></a>
```

Comment:
- [ ] **[REQUIRED CHANGES]** As the previous reviewer requested, your hamburger icon is not loading properly. If you look closely, you will notice that the file extension is `.pnggit`.

Reply:
Done, the hamburger icon button-mobile had its path changed to the correct one.